.github/workflows/publish.yaml

```
@@ -17,12 +17,12 @@ jobs:
       - name: Check libraries
         uses: canonical/charming-actions/check-libraries@1.0.0
```

```suggestion
        uses: canonical/charming-actions/check-libraries@2.0.0-rc
```

I'd recommend bumping these versions up to the latest release candidate/stable 2.0 when it releases as changes to Charmcraft will make these older actions less palatable :)

---

Makes sense, thanks! :)

Will have to update all the other projects as well.